package/yast2-theme.spec

```diff
@@ -82,11 +82,13 @@ yast2-branding-openSUSE package.
 %package SLE
 Summary:        YaST2 - SLE Theme
 Group:          System/YaST
-Provides:       yast2-theme-NLD = 0.4.6
 Provides:       yast2_theme = %{version}
-Obsoletes:      yast2-theme-NLD <= 0.4.5
 Conflicts:      yast2-theme-openSUSE
 Conflicts:      yast2-theme-openSUSE-Oxygen
+Conflicts:      yast2-branding-openSUSE
+Conflicts:      yast2-branding-openSUSE-Oxygen
+Obsoletes:      yast2-branding-openSUSE
+Obsoletes:      yast2-branding-openSUSE-Oxygen
```

Comment:
why also this obsoletes? Is it needed?

Reply:
I think yes, e.g. yast2-branding-openSUSE and yast2-branding-openSUSE-Oxygen is installed and you migrate to SLES. So both package will be deleted without informing the user. Otherwise the user will get an error message that yast2-branding-openSUSE will be deleted bug yast2-branding-openSUSE-Oxygen still needs it.

Reply:
ok, then fine for me.